content/site.json

```diff
@@ -22,7 +22,7 @@
       },
       {
         "name": "Catalogue",
-        "link": "/search?sort=_score&sortOrder=desc&text="
+        "link": "/search?sort=_score&sortOrder=desc&isCompliant=true&text="
```

Comment:
Okay, where can the user update this filter?

Reply:
The user applies the filter clicking on a button. This change was a mistake and I already fixed it.